shared/util/second-timer.js

```diff
@@ -21,6 +23,25 @@ class Ticker {
   refs: Array<Ref> = []
   intervalID: ?IntervalID
 
+  constructor() {
+    this._setupDebugging()
+  }
+
+  _setupDebugging = () => {
+    if (!__DEV__) {
+      return
+    }
+
+    if (printOutstandingTimerListeners) {
+      // Print any listeners to this periodically
+      setInterval(() => {
```

Comment:
you can actually do this in your one timer

Reply:
done :+1: